### ISSUE TITLE
atomic-reactor fails on RHEL7 because of python-websocket-client version

### ISSUE THREAD

**Issue by maxamillion** [projectatomic/atomic-reactor#262]
Built from master in [COPR](https://copr.fedoraproject.org/coprs/maxamillion/atomic-reactor/), `atomic-recator` fails on RHEL7

```
atomic-reactor create-build-image --reactor-tarball-path /usr/share/atomic-reactor/atomic-reactor.tar.gz /usr/share/atomic-reactor/images/dockerhost-builder buildroot
Traceback (most recent call last):
  File "/bin/atomic-reactor", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 3011, in <module>
    parse_requirements(__requires__), Environment()
  File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 626, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: websocket-client>=0.32.0
```

The version of python-websocket-client is pretty old:
```
yum info python-websocket-client
Loaded plugins: product-id, subscription-manager
Installed Packages
Name        : python-websocket-client
Arch        : noarch
Version     : 0.14.1
Release     : 108.el7
Size        : 148 k
Repo        : installed
From repo   : rhel7-extras
Summary     : WebSocket client for python
URL         : http://www.docker.com
License     : LGPLv2
Description : python-websocket-client module is WebSocket client for python. This
            : provides the low level APIs for WebSocket. All APIs are the synchronous
            : functions.
            : 
            : python-websocket-client supports only hybi-13.
```

**Comment by TomasTomecek** [comment]
Oh boy. There's not much we can do. We can provide updated `python-websocket-client`, but that's not really a solution, just a workaround. This needs to be solved in rhel. We could clone [this](https://bugzilla.redhat.com/show_bug.cgi?id=1249651) for websocket package.

**Comment by bkabrda** [comment]
I already reported a problem with version of python-requests [1] and hopefully it'll be fixed in the docker-python package. Since both docker-python and python-websocket-client are built from `docker.src.rpm` in RHEL 7 (ugly!), I suggest you leave another comment at the referenced bug about this. Hopefully the maintainer can fix both at the same time. Otherwise, as Tomas said, there's not much we can do about dependencies packaged in a wrong way. (Well, you can package correct version of python-websocket-client in your COPR repo, but I guess you already know that you can do that as a workaround).

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1249651

**Comment by mmilata** [comment]
Closing as referenced BZ is closed and `python-websocket-client-0.32.0-116.el7.noarch` is now in RHEL7. Please reopen if this bug is still not fixed.
